Doc/library/dis.rst

```diff
@@ -847,24 +851,59 @@ the more significant byte last.
 
 .. opcode:: CALL_FUNCTION_VAR (argc)
 
-   Calls a function. *argc* is interpreted as in :opcode:`CALL_FUNCTION`. The
-   top element on the stack contains the variable argument list, followed by
-   keyword and positional arguments.
+   Calls a callable object, similarly to :opcode:`CALL_FUNCTION`.
+   *argc* represents the number of keyword and positional
+   arguments, identically to :opcode:`CALL_FUNCTION`.
+   The top of the stack contains an iterable object containing
+   additional positional arguments.
+   Below that are keyword arguments (if any), positional arguments (if any)
+   and a callable object, identically to :opcode:`CALL_FUNCTION`.
+   Before the callable object is called, the iterable object
+   is "unpacked" and its contents are appended to the positional
+   arguments passed in.
+   The iterable object is ignored when computing
+   the value of ``argc``.
+
+   .. versionchanged:: 3.5
```

Comment:
Presumably you've added this `versionchanged` note in the 2.7 docs for the sake of porting.  I can see the value of that, but it may not be the right thing to do.  Perhaps I've misunderstood the reason?

Regardless, I'm not sure that such a "forward-referencing" `versionchanged` note is correct here.  It's not something that we do anywhere else in the docs.  From what I understand, that's what the [porting section of the "What's New" doc](https://docs.python.org/3/whatsnew/3.5.html#porting-to-python-3-5) is for.  Otherwise the docs would already be full of forward-referencing `versionchanged` notes, the devguide would mention it, and folks making changes would be responsible for updating the docs for all the past versions with such notes.

FWIW, I don't see a mention of `CALL_FUNCTION_VAR` in the porting section for 3.5, so it would probably be worth adding that.  I'm sure you're already aware, but remember that the 3.5 "What's New" doc is in each branch from 3.5 onward.  I'll open an issue for this.